smbackend_turku/tests/test_addresses_import.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from django.test import TestCase
+from django.conf import settings
+
+from munigeo.models import Address, Municipality
+from django.contrib.gis.geos import Point
+
+from smbackend_turku.importers.addresses import AddressImporter
+
+
+class AddressImportTestCase(TestCase):
```

Comment:
I would prefer to change this from Testcase to Pytest to be consistent with other tests in this project.

Otherwise, very nice work! 👍